llvm/unittests/ProfileData/MemProfTest.cpp

```diff
@@ -807,4 +807,38 @@ TEST(MemProf, YAMLParser) {
   EXPECT_THAT(Record.CallSiteIds,
               ElementsAre(hashCallStack(CS3), hashCallStack(CS4)));
 }
+
+TEST(MemProf, YAMLWriterFrame) {
+  Frame F(12, 34, 56, true);
+
+  std::string Out;
+  llvm::raw_string_ostream OS(Out);
+  llvm::yaml::Output Yout(OS);
+  Yout << F;
```

Comment:
You could refactor this out into a little helper (templated to the type being serialized) that can be reused across the tests.

Reply:
Done.